README.md

```diff
@@ -1,70 +1,76 @@
-# Getting Started with Create React App
+# Space Travelers
 
```

Comment:
- [ ] Your README looks nice but I think adding a screenshot of your project would make it look nicer